src/log.js

```diff
@@ -26,88 +26,6 @@ function decapitalize(str) {
   return str && str[0].toLowerCase() + str.slice(1)
 }
 
-/**
```

Comment:
I cleaned up logger a bit because we had a few lint issues connected with these functions. And we don't use them anyway.